cfgov/v1/models/feedback.py

```diff
@@ -3,13 +3,9 @@
 from django.utils.encoding import force_text
 from django.utils.timezone import localdate
 
-from backports import csv
-
+from wagtail.core.models import Page
 
-try:
-    from wagtail.core.models import Page
-except ImportError:  # pragma: no cover; fallback for Wagtail < 2.0
-    from wagtail.wagtailcore.models import Page
+from backports import csv
```

Comment:
I think we can just import the standard Python 3 csv library here and drop backports.csv from requirements, but that's fort another PR.

Reply:
I did not include these Python 3 changes in this PR.